Dockerfile

```
@@ -19,7 +17,9 @@ RUN addgroup ${GROUP} &&\
 COPY run.sh /app/
 RUN chmod a+x /app/run.sh
 
-RUN apk --no-cache update &&\
+RUN apk upgrade &&\
+    apk --no-cache update &&\
+    apk add bash &&\
     apk add --update --no-cache curl py-pip gnupg py-setuptools ca-certificates groff less &&\
```

```suggestion
    apk add --no-cache bash curl py-pip gnupg py-setuptools ca-certificates groff less &&\
```

Doing `upgrade` before `update` won't do anything useful, because you need the updated list of packages before you can upgrade them.

I'm not entirely sure if `upgrade` will actually do anything considering the version peg of `FROM alpine:3.14`. Maybe we should just use `FROM alpine:3.16` (or maybe even `alpine:3`?)